RevenueCatUI/CustomerCenter/ViewModels/ManageSubscriptionsViewModel.swift

```diff
@@ -66,10 +66,12 @@ class ManageSubscriptionsViewModel: ObservableObject {
     }
 
     init(screen: CustomerCenterConfigData.Screen,
-         subscriptionInformation: SubscriptionInformation) {
+         subscriptionInformation: SubscriptionInformation,
+         refundRequestStatusMessage: String? = nil) {
```

Comment:
I wasn't entirely sure about this, but from what I can tell this constructor is only used in previews so I thought it'd be okay. Happy to revert of course.

Reply:
yeah it's fine, these constructors for previews are always very not-ideal

Reply:
Though this just now but we could wrap this constructor in a `#if DEBUG` macro, so we make sure it's not used in release builds... NABD though.